test/web3/schema.json

```diff
@@ -0,0 +1,163 @@
+{
+    "params": "0xb3b20624f8f0f86eb50dd04688409e5cea4bd02d700bf6e79e9384d47d6a5a35",
```

Comment:
I think it would be good to give descriptive names to these params. For example, `"accountAddressArray"` instead of `"params2"`